conda/faiss/build-pkg.sh

```diff
@@ -21,6 +21,7 @@ cmake --build _build_python_${PY_VER}_avx2 -j $CPU_COUNT
 # Build vanilla version (no avx).
 cmake -B _build_python_${PY_VER} \
       -Dfaiss_ROOT=_libfaiss_stage/ \
+      -DFAISS_OPT_LEVEL=avx2 \
```

Comment:
You might want to switch which build has avx2, to remain consistent with the comments.

Reply:
My bad, I got the order wrong.